google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/vcs/CloneCloudRepositoryDialog.java

```diff
@@ -267,15 +268,25 @@ protected JComponent createCenterPanel() {
   private void paintSelectionOk() {
     parentDirectory.setBackground(Color.getColor("ECECEC"));
     parentDirectoryLabel.setForeground(Color.BLACK);
+    directoryNameLabel.setForeground(Color.BLACK);
   }
 
   /**
-   * Activates when a user selection is incorrect.
+   * Activates when a user selection of parent directory is incorrect.
    * <p></p>
    * Paints the "Parent Directory" label and textbox background red.
    */
-  private void paintSelectionError() {
+  private void paintParentDirectorySelectionError() {
+    paintSelectionOk();
     parentDirectory.setBackground(Color.RED);
     parentDirectoryLabel.setForeground(Color.RED);
   }
+
+  /**
+   * Paints the directory selector red when in error.
+   */
+  private void paintDirectorySelectionError() {
+    paintSelectionOk();
+    directoryNameLabel.setForeground(Color.RED);
```

Comment:
nit: maybe you also want to set the directoryName background to red, so the error renders similarly to the parent directory one?